roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml

```diff
@@ -25,7 +25,7 @@
   become: false
 
 - name: generate htpasswd file for hawkular metrics
-  local_action: htpasswd path="{{ local_tmp.stdout }}/hawkular-metrics.htpasswd" name=hawkular password="{{ hawkular_metrics_pwd.content | b64decode }}"
+  local_action: htpasswd crypt_scheme=ldap_sha1 path="{{ local_tmp.stdout }}/hawkular-metrics.htpasswd" name=hawkular password="{{ hawkular_metrics_pwd.content | b64decode }}"
```

Comment:
Shouldn't we make the scheme configurable by a variable? This way everyone will get SHA1, but SHA1 is considered insecure because it doesn't use salt (at least `man htpasswd` says this). Which is kinda funny given that it's allowed by FIPS.

Reply:
Thanks for the review!

Yeah that would be good, I'll add that configuration parameter with MD5 as a default

I think the fact that FIPS allows SHA1 is because the algorithm itself is more secure that MD5, but in the particular case of hwpasswd they don't salt the password for SHA1.